### DISCUSSION TITLE
How to. change theme color for apollo theme, never effect on init process

### DISCUSSION THREAD

**Discussion by mallka** [primefaces/community#2700] [PrimeVue Templates] [answered]
Hello sir,

Regarding Apollo verson: v6.0.0, I have a trouble that how to change the color ?

```
//. page: src/layout/composables/layout.js

import { computed, reactive, ref, watch } from 'vue';
const layoutConfig = reactive({
    preset: 'Aura',
    primary: 'blue',
    surface: null,
    darkTheme: false,
    menuMode: 'horizontal',
    menuTheme: 'colorScheme'
});
```

and then I found the primary color never took effect, but the menuMode is working fine.
Could you let me know how I can change it or where I can found the document for it ?

Looking foreward to your reply.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

Official documentation: https://primevue.org/theming/styled/#primary

**Comment by mallka** [comment]
Thank you . 
Solved.
